### ISSUE TITLE
Missing reports/broken profiles for individuals with empty license object

### ISSUE THREAD

**Issue by jnccloud** [Project-Sloth/ps-mdt#59]
**Symptoms**

There have been a multitude of players who seem to have an issue with their licenses being erased. When this happens, all licenses and data disappear from the MDT for these users, and you can no longer make or edit that users profile. Vehicles, houses, etc., none of these populate.

**Research**
I have successfully restored these profiles by modifying their metadata via SQL and replacing it with the expected value to be in the database:
`"licences":{"business":false,"driver":false,"weapon":true,"pilot":false}` (valid profile, no issues in MDT)
`"licences":{}` (this users profile will be broken, as I assume because at least one license is not in the metadata)

I have also seen multiple references to the English term "licences" but in the MDT, [some functions refer to it as "licenses".](https://github.com/Project-Sloth/ps-mdt/search?q=licenses) I feel like this needs to be standardized just to reduce the confusion in the future; I'm assuming to "licences" as a multitude of other resources expect this term to be in the player metadata.

The major concern with this is that when this happens, it breaks gun licenses, city hall, etc., for that person. They can no longer obtain or purchase anything that requires a license. I've instructed LEO to use the `/grantlicense` command in the meantime.

My gut tells me that something doesn't happen as expected when giving or removing a license from someone via the MDT, and that's when the empty object is created.

**Comment by jnccloud** [comment]
tl;dr - an empty license object is in the player's metadata, and it breaks the profile when you load it via the MDT.

Sorry for the long issue! Just wanted to give as much detail as I had.
